recipes/rdma-core/all/conanfile.py

```diff
@@ -0,0 +1,133 @@
+import os
+import re
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout, CMakeDeps
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import get, copy, rmdir, load, save, replace_in_file
+from conan.tools.gnu import PkgConfigDeps
+
+required_conan_version = ">=1.53.0"
+
+class PackageConan(ConanFile):
+    name = "rdma-core"
+    description = ("RDMA core userspace libraries and daemons. "
+                   "Provides userspace components for the Linux Kernel's drivers/infiniband subsystem.")
+    license = ("GPL-2.0", "Linux-OpenIB", "BSD-2-Clause")
```

Comment:
I think this is GPL-2.0-only.

```suggestion
    license = ("GPL-2.0-only", "Linux-OpenIB", "BSD-2-Clause")
```